sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -335,7 +335,7 @@ object FunctionRegistry {
     val df = clazz.getAnnotation(classOf[ExpressionDescription])
     if (df != null) {
       (name,
-        (new ExpressionInfo(clazz.getCanonicalName, name, df.usage(), df.extended()),
+        (new ExpressionInfo(clazz.getCanonicalName, name, df.usage(), df.extended().stripMargin),
```

Comment:
Annotation argument needs to be a constant. Thus, I called stripMargin when building ExpressionInfo. I am not sure if this is a right way. If so, should we call stripMargin for df.usage() too? 

If we do not care indenting, there exists another solution: 
```scala 
  extended = """> SELECT _FUNC_(array("b", "d", "c", "a"));
4""")
```
Or do you have a better way for adding a newline marker between triple quotes? Thank you! @marmbrus